content/about/at-support-tables/at-support-tables.html

```diff
@@ -0,0 +1,122 @@
+      This page explains how to interpret and use the assistive technology support summaries.
+    </p>
+
+    <section id="purpose">
+      <h2>Purpose of AT Support Tables</h2>
+      <p>
+        The purpose of the support tables is to provide an actionable summary of the interoperability tests performed by the <a href="http://aria-at.w3.org/">ARIA-AT project</a>.</p>
+    </section>
+
+    <section id="meaning">
+      <h2>Meaning of Support Levels</h2>
+      <p>
+        The assistive technology support tables present two percentages for each assistive technology and browser combination that have been tested: "Must-Have Behaviors" and "Should-Have Behaviors".
+         A behavior designated as “Must-Have" is essential; if not provided, users could be blocked from using the UI element.
+        Failure to provide a “Should-Have” behavior could impede users.
+        Learn more about ARIA-AT’s
+        <a href="https://github.com/w3c/aria-at/wiki/Glossary#assertion-priority">definitions of Must and Should on the project wiki</a>.
+        </p>
+
+      <section>
+        <h3>Examples of Must-Have Behaviors</h3>
+        <ul>
+          <li>Convey the name of a radio button.</li>
+          <li>Convey the state of a checked radio button.</li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>Examples of Should-Have Behaviors</h3>
+        <ul>
+          <li>Convey the position of a radio button in a radio group, e.g., the button is 1 of 3.</li>
+          <li>Convey the number of radio buttons in a radio group.</li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>Important Constraints</h3>
+        <ul>
+          <li>Unless otherwise noted, all testing is done using the assistive technology vendor's default configuration of an assistive technology.</li>
+          <li>
+            ARIA-AT interoperability  tests do not prescribe exactly how to satisfy a need.
+            For example, they do not specify exactly what a screen reader should speak.
+            Two different screen readers may convey the same information in different ways.
+          </li>
+        </ul>
+      </section>
+    </section>
+
+    <section id="recommendations">
+      <h2>Recommendations</h2>
+
+      <section>
+        <h3>Don’t Code to the Bugs</h3>
+        <p>
+          ARIA-AT is working with assistive technology vendors to increase their support levels.
+          This means that assistive technologies that align with ARIA-AT interoperability tests will change over time.
+          Exercise caution when implementing a pattern where support levels are less than 100%.
+          Avoid modifying code to accommodate an assistive technology failure unless you are confident that the modification will:
+        </p>
+        <ul>
+          <li>Not negatively affect the experience when using assistive technologies that provide 100% support.</li>
```

Comment:
@mcking65 Would it be clearer if we update "that provide 100% support" as "that provide 100% **must-have** support"?

Reply:
It does not matter if it is a must or should that is being accomodated. However, your question prompted me to clarify by avoiding the need to use support levels as the factor that needs to be considered when making this judgment.